vars/checkCache.groovy

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/groovy
```

Comment:
The `checkCache` name is misleading as this function will actually restore cached data if it exists.